chat-plugins/scavengers.js

```diff
@@ -928,7 +928,7 @@ let commands = {
 		points = parseInt(points);
 
 		if (!targetId || targetId === 'constructor' || targetId.length > 18) return this.errorReply("Invalid username.");
-		if (!points || points < 0) return this.errorReply("Points must be an integer greater than 0.");
+		if (!points || points < 0 || points === Infinity) return this.errorReply("Points must be a non-Infinity integer greater than 0.");
```

Comment:
It's possible to add two non-Infinity values and yet end up with Infinity. Wouldn't it be simpler to just put some cap on points possible to earn? Getting 34829521891282195912953269328683 points doesn't sound all that reasonable.

```
num = parseInt(1 + '7'.repeat(308))
1.7777777777777777e+308
num + num
Infinity
```